stdlib/public/Concurrency/Actor.cpp

```diff
@@ -1822,7 +1851,9 @@ bool DefaultActorImpl::unlock(bool forceUnlock)
 
       if (newState.isScheduled()) {
         // See ownership rule (6) in DefaultActorImpl
-        scheduleActorProcessJob(newState.getMaxPriority());
+        // FIXME: we have to respect task executor here as well
+        //        - how to get the right value to the unlock?
+        scheduleActorProcessJob(newState.getMaxPriority(), TaskExecutorRef::undefined());
```

Comment:
This is a TODO rather, I'm not sure if we need it here. We can get the task executor out of a task via the preference record, so it'd be possible. It probably is necessary, so someone unlocking but we have more tasks to run, and we may need to respect preference... I need to read through this some more, hints welcome though